.github/workflows/native-build.yml

```
@@ -0,0 +1,81 @@
+          java-version: '24'
+          distribution: 'graalvm-community'
+          native-image-job-reports: 'true'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v5
+
+      - name: ネイティブイメージをビルド
+        run: ./gradlew :backend:nativeCompile
+
+      - name: ネイティブバイナリの healthz スモークテスト
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2046:warning:1:8: Quote this to prevent word splitting

(shellcheck)